### PR TITLE
Update voter tutorial to Starport v0.17

### PR DESCRIPTION
**Pull request by toschdev**
Update voter tutorial to Starport v0.17

For easier review, text and code split in two PRs:
text: #766
code: #767

### CODE REVIEW

File: voter/index.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 order: 1
-description: Create a simple blockchain poll application, add types and messages, and design the front-end app.
+description: Create a blockchain poll application, add types and messages, and design the front-end app.
 ---
 
 # Polling app
 
-This tutorial creates a simple blockchain poll application.
+This tutorial creates a blockchain poll application.
```
Comment (barriebyron) [inline voter/index.md:8]: :praise:  I see you are embracing the Digital Ocean philosophy of not using "simple" good job!

File: voter/index.md [hunk 2]
```diff
@@ -66,17 +66,17 @@ Design the app so that you can:
 
 Use Starport to scaffold the blockchain app and the voting module.
 
-**Important** In the code examples throughout this tutorial, when you see `username` be sure to substitute it with _your username_. You need to do this in some of the Vue and REST API examples later. The app end user in this tutorial is alice. {synopsis}
+**Important** In the code examples throughout this tutorial, when you see `cosmonaut` be sure to substitute it with _cosmonaut_. You need to do this in some of the Vue and REST API examples later. The app end user in this tutorial is alice. {synopsis}
```
Comment (barriebyron) [inline voter/index.md:69]: 🥇  Fabulous! We are cosmonauts.

File: voter/index.md [hunk 3]
```diff
@@ -476,7 +413,7 @@ Now, restart the application. Remember to use the `--reset-once` flag to recogni
 starport serve --reset-once
 ```
 
-Each time you reset the application state, all of the data from your previously created state is saved.
+Each time you reset the application state, all of the data from your previously created state is not saved.
```
Comment (barriebyron) [inline voter/index.md:416]: oof! I didn't know this. Not saving data is expected behavior?

**Review by barriebyron — APPROVED** [review]
@toschdev great job on these many important changes. I can't verify the technical code, but everything else looks good to me.

File: voter/index.md (hunk 3 above)
Comment (toschdev) [reply voter/index.md:416]: Default behavior is saving the data. When using `--reset-once` state though, the state is not saved.

**Pull request merged**